src/WebJobs.Extensions.DurableTask/FunctionState.cs

```diff
@@ -12,6 +12,7 @@ internal enum FunctionState
         Completed,
         Terminated,
         Failed,
-        ExternalEventRaised
+        ExternalEventRaised,
+        TimerExpired
```

Comment:
>TimerExpired [](start = 8, length = 12)

Is there a way to write test for this new state?  Sorry I can't seem to find it.

Reply:
Would TestHelpers.GetLogs_TimerExpiration be the right place for this?

---
In reply to: [154245997](https://github.com/Azure/azure-functions-durable-extension/pull/103#discussion_r154245997) [](ancestors = 154245997)

Reply:
yes

---
In reply to: [154248091](https://github.com/Azure/azure-functions-durable-extension/pull/103#discussion_r154248091) [](ancestors = 154248091,154245997)